### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/deployment/index.md` and `~/deployment/versions.md`

### PR DESCRIPTION
**Pull request by pabloprx**
Added translation.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit aacc3329187e6614692c94d4e9b37aa6f43957d0 at: https://64889496bfd5811f4737a309--fastapi.netlify.app

**Review by piaria — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 34c71674c375dfe90d11caf170d6540d5798e29a at: https://648b7a768c49450b2a22e8a7--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit ca61686c854ce64f8764f29f152b298b5ffca9fc at: https://648b8243a5203d10ad812d76--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit 39533732ff37ef47ecdccba3569a99af2ce40510 at: https://648c7f95a3af7c297d59901d--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit a261471a8119e8d4a2a51528059b6d1cb3177cb4 at: https://648c817c98daf828b89d6b37--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit 827b33b8e4d39bcbf6b4e14e281086d89fbd1c27 at: https://648c8757f0fb642c6e285346--fastapi.netlify.app

**Comment by pabloperezmoya** [comment]
My bad on a rebase, I shouldn't rebase on changes not related to this translate work.

**Review by alejsdev — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7813af5c14ab6689eab3b6ad9b30f51b65960ff7 at: https://80bb470b.fastapitiangolo.pages.dev

File: docs/es/docs/deployment/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+**Desplegar** una aplicación significa realizar una serie de pasos para hacerla **disponible para los usuarios**.
+
+Para una **API web**, normalmente implica ponerla en una **máquina remota**, con un **programa de servidor** que proporcione un buen rendimiento, estabilidad, etc, para que sus **usuarios** puedan **acceder** a la aplicación de manera eficiente y sin interrupciones o problemas.
+
+Esto difiere en las fases de **desarrollo**, donde estás constantemente cambiando el código, rompiéndolo y arreglándolo, deteniendo y reiniciando el servidor de desarrollo, etc.
+
+## Estrategias de despliegue
+
+Existe varias formas de hacerlo dependiendo de tu caso de uso específico y las herramientas que uses.
```
Comment (tiangolo) [inline docs/es/docs/deployment/index.md:15]:
```suggestion
Existen varias formas de hacerlo dependiendo de tu caso de uso específico y las herramientas que uses.
```

File: docs/es/docs/deployment/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Puedes **desplegar un servidor** tú mismo usando un conjunto de herramientas, puedes usar **servicios en la nube** que haga parte del trabajo por ti, o usar otras posibles opciones.
+
+Te enseñaré algunos de los conceptos principales que debes tener en cuenta al desplegar aplicaciones hechas con **FastAPI** (aunque la mayoría de estos conceptos aplican para cualquier otro tipo de aplicación web).
+
+Podrás ver más detalles de las técnicas que tengo en mente en las próximas secciones.✨
```
Comment (tiangolo) [inline docs/es/docs/deployment/index.md:21]:
```suggestion
Podrás ver más detalles para tener en cuenta y algunas de las técnicas para hacerlo en las próximas secciones.✨
```

File: docs/es/docs/deployment/versions.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Puedes crear aplicaciones listas para producción con **FastAPI** ahora mismo (y probablemente lo has estado haciendo por algún tiempo), solo tienes que asegurarte de usar la versión que funciona correctamente con el resto de tu código.
+
+## Fijar la versión de `fastapi`
+
+Lo primero que debes hacer en tu proyecto es "fijar" la última versión específica de **FastAPI** que sabes que funciona bien con tu aplicación.
+
+Por ejemplo, digamos que estás usando la versión `0.45.0` en tu aplicación.
+
+Si usas el archivo `requirements.txt` puedes especificar la versión con:
+```txt
```
Comment (tiangolo) [inline docs/es/docs/deployment/versions.md:18]:
```suggestion

```txt
```

File: docs/es/docs/deployment/versions.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+fastapi==0.45.0
+```
+
+esto significa que usarás específicamente la versión `0.45.0`.
+
+También puedes fijar las versiones de esta forma:
+```txt
```
Comment (tiangolo) [inline docs/es/docs/deployment/versions.md:25]:
```suggestion

```txt
```

File: docs/es/docs/deployment/versions.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+Siguiendo las convenciones de *Semantic Versioning*, cualquier versión por debajo de `1.0.0` puede potencialmente tener <abbr title="cambios que rompen funcionalidades o compatibilidad">*breaking changes*</abbr>.
+
+FastAPI también sigue la convención de que cualquier cambio hecho en una <abbr title="versiones de parche">"PATCH" version</abbr> es para solucionar errores y <abbr title="cambios que no rompan funcionalidades o compatibilidad">*non-breaking changes*</abbr>.
+
+!!! tip
+    El <abbr title="parche">"PATCH"</abbr> es el último número, por ejemplo, en `0.2.3`, la <abbr title="versiones de parche">PATCH version</abbr> es `3`.
+
+Entonces, deberías fijar la versión así:
+```txt
```
Comment (tiangolo) [inline docs/es/docs/deployment/versions.md:47]:
```suggestion

```txt
```

**Comment by tiangolo** [comment]
Thanks @pabloperezmoya! ☕ 

We just added a few tweaks and now it's ready. 🍰 

Thanks for the help @alejsdev and @piaria! 🍪

**Pull request merged**